tests/emukit/bayesian_optimization/test_local_penalization.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from scipy.optimize import check_grad
+
+from emukit.bayesian_optimization.acquisitions.local_penalization import LocalPenalization
+from emukit.core.interfaces import IModel
+
+TOL = 1e-6
+
+
+def test_penalization_function():
```

Comment:
might want to call this 'test_penalization_function_shape()' so that it matches the test function name below

Reply:
Will do